packages/apollo-collaboration-server/src/features/features.service.ts

```diff
@@ -196,19 +212,38 @@ export class FeaturesService {
 
     const { assembly } = exportDoc
     const refSeqs = await this.refSeqModel.find({ assembly }).exec()
+    const refSeqIds = refSeqs.map((refSeq) => refSeq._id)
+    let printFasta = true
 
     const headerStream = new Readable({ objectMode: true })
+    const sequenceStream = new Readable({ objectMode: true })
+
+    // Implement the _read() method for the custom stream. This is just a placeholder; we can leave it empty
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    headerStream._read = function () {}
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    sequenceStream._read = function () {}
+
     headerStream.push('##gff-version 3\n')
     for (const refSeqDoc of refSeqs) {
       headerStream.push(
         `##sequence-region ${refSeqDoc.name} 1 ${refSeqDoc.length}\n`,
       )
+      for await (const doc of this.refSeqChunksModel
+        .find({ refSeq: refSeqDoc.id })
+        .sort({ n: 1 })
+        .cursor()) {
+        printFasta ? sequenceStream.push('##FASTA\n') : null
+        sequenceStream.push(`>${refSeqDoc.name} ${refSeqDoc.description}\n`)
```

Comment:
If there is no description, this should not print a space after the name so there aren't any trailing spaces.

Reply:
Fixed